ui/app/models/pki/role.js

```diff
@@ -119,24 +119,7 @@ export default class PkiRoleModel extends Model {
     label: 'Signature bits',
     subText: `Only applicable for key_type 'RSA'. Ignore for other key types.`,
     defaultValue: 0,
-    possibleValues: [
-      {
-        value: 0,
-        displayName: 'Defaults to 0',
-      },
-      {
-        value: 256,
-        displayName: '256 for SHA-2-256',
-      },
-      {
-        value: 384,
-        displayName: '384 for SHA-2-384',
-      },
-      {
-        value: 512,
-        displayName: '512 for SHA-2-5124',
-      },
-    ],
+    possibleValues: [0, 256, 384, 512],
```

Comment:
updated to match designs, can revert if design says otherwise

Reply:
Is it sufficient to display the values rather than the strings that were defined previously with `displayName` then?

Reply:
Yeah - heard back from design and the values here are actually preferred over the displayNames